### ISSUE TITLE
Keymap for "Select to Enclosing Bracket"

### ISSUE THREAD

**Issue by ryrobes** [zed-industries/zed#12772]
### Check for existing issues

- [X] Completed

### Describe the feature

Jumping between brackets is great - but often I like to use shift & select the entire bracket as well (for lisp type languages, removing a whole form, etc). So it would be great to have, in addition to "editor::MoveToEnclosingBracket", also a simple "editor::SelectToEnclosingBracket" -  just as we have "SelectToWord*" etc.

**Comment by vladmoroz** [comment]
Agree, VS Code has a command called `editor.action.selectToBracket`

**Comment by shotanue** [comment]
Nice, I also want that keymap.

**Comment by diegorojasj** [comment]
Me too. I frequently use this keymap :'v

**Comment by bondiano** [comment]
I found some alternative using `editor::SelectLargerSyntaxNode`. It's not the same but u can try to use it too

**Comment by aurexav** [comment]
> I found an alternative using `editor::SelectLargerSyntaxNode`. It's not the same, but you can try to use it too.

This is awesome. This can handle the `<>`.
